packages/snjs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@standardnotes/snjs",
-  "version": "2.0.76",
+  "version": "2.0.77",
```

Comment:
semver nitpick (to ignore if this is a bugfix or we don't care): if this change is considered a new (backwards-compatible) feature (as `feat` in the commit suggests), the minor component of the version should be bumped rather than the patch component (which is for backwards-compatible bugfixes).

Reply:
Increasing anything but the patch version gives me anxiety 😥

Reply:
Looking at [the summary of semver](https://semver.org/#summary) to guide your decision may ease the anxiety. ;)

Anyway, adhering to semver is not strictly necessary for internal code. And in fuzzy cases it may indeed be difficult to make these decisions. I myself am not sure how to classify this change, so just pointing out, based on the commit message that minor might be more appropriate here.

In the end it's a matter of agreement between the library authors and users what the version means and what to expect when it changes.

Reply:
A good argument for being more aggressive in increasing the minor and major components: this way you err on the safe side -- the worst that happens is you communicate that you introduced a bigger or a breaking change when you didn't. This is better than the reverse: communicating that no breaking change was introduced when there was.

A node-specific argument for adhering to semver is that it is a common (and expected) practice -- `npm`s dependency, `node-semver` expects the `"version"` field from `package.json` to be at least formatted according to semver. Enforcing meaning is of course impractical.

So probably a safe default is to adhere to semver unless explicitly following a different policy.

Reply:
An argument I would make in favor of following semver: we have sometimes been stuck wanting to bundle an SNJS fix into a client, but there was a bigger feature in the commit tree as well that we didn't want to include. Had we used semver it would have been easier to branch off `master` and publish a minor version with only the fixes we needed.

Reply:
Agreed. But I would say this really isn't a major change nor a backward incompatible one. We don't rely on the functionality we're disabling. Just a preventative measure.

Reply:
@mobitar sounds like increasing patch is fine in this case then -- it fixes a (potential) issue in a backwards-compatible manner (assuming nothing will break), and does not add any functionality

what may be considered confusing is the `feat` in the commit, which suggests a feature -- the intended meaning in the [Conventional Commits spec](https://www.conventionalcommits.org/en/v1.0.0/) is:

> feat: a commit of the type feat introduces a new feature to the codebase (this correlates with MINOR in Semantic Versioning).

now do we want to start using these in line with these intentions or do we agree that they only have provisional meaning for us (as seems to be the case so far)?

Reply:
Not sure, but will think harder about it next time.